Project 4/README.md

```diff
@@ -44,4 +44,48 @@ You're probably thinking, "why would I submit a level 8 kata if they're not wort
 
 It is your responsibility and the responsibility of your peers reviewing your submission in PR to determine whether your submission is ranked appropriately. In the event that consensus is reached that your kata is ranked inappropriately, you must work with your peers to revise the submission so that it is either more or less challenging, accordingly. You are not permitted to submit new problems with different strengths after PRs are open, but must instead revise your PRs. So, think hard about how challenging your submission is. 
 
-There is one other option for those desiring a different sort of challenge. If you provide alongside your SPARQL submission a translation of the same problem into SQL, complete with documentations, solution, etc. then you may receive half points extra at that kata level (rounded up). For example, if you submit a SPARQL problem that is kata rank 1 and also submit a SQL version of that same problem, you  will receive 35+18=53 points. 
+There is one other option for those desiring a different sort of challenge. If you provide alongside your SPARQL submission a translation of the same problem into SQL, complete with documentations, solution, etc. then you may receive half points extra at that kata level (rounded up). For example, if you submit a SPARQL problem that is kata rank 1 and also submit a SQL version of that same problem, you  will receive 35+18=53 points.
+
+
+
+QUESTION 1 - Kata 8
+Write a SPARQL query that provides all of the types of “things” under which Barack Obama is classified, according to DBpedia.
+
+TIPS: 
+Use {dbr:Barack_Obama} as the Subject.
+Use {rdf:type} as the Predicate.
+
+ANSWER
+PREFIX dbr: http://dbpedia.org/resource
+PREFIX rdf: http://www.w3.org/1999/02/22-rdf-syntax-ns#
+
+SELECT ?Concept dbr:Barack_Obama
+WHERE 
+{
+    dbr:Barack_Obama a  ?Concept .
+}
+
+
+
+QUESTION 2 - Kata 6
+Not all information on the internet is accurate information. To give an example of this, write a SPARQL query to check if Barack Obama is a city according to DBpedia. Limit the results to 50 and order them alphabetically. In addition, ensure there are no duplicate results and ensure that the results are case-independent.
```

Comment:
I think it is better to change your question description to "find at least 50 types to which Obama belongs, and they should include 'cit' in their labels". After all, the original description "check if Barack Obama is a city according to DBpedia" sounds like a YES/NO question. If so, it can be solved by an ASK query.

Reply:
Hmmmm, there wouldn't be 50 types, because there are only 3 results, but I get what you mean. I'll try to look into the ASK query and see if I can get something done with that.